src/PowerShell/Microsoft.WinGet.Client.Engine/Microsoft.WinGet.Client.Engine.csproj

```diff
@@ -98,6 +97,7 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(TargetFramework)' == '$(DesktopFramework)'">
+    <LangVersion>10</LangVersion>

```

Comment:
Is the move from global to DesktopFramework only intentional?

Reply:
Yes, as the embedded code requires a higher version.